### PR TITLE
PEPPER-833 RGP new kit name

### PR DESCRIPTION
**Pull request by pegahtah**
PEPPER-833 Adding `kitTypeDisplayName` for shipping.model.ts and changing the display in html to show type first by kitDisplayName and if not, by the kitType to accomodate RGP's new name for BLOOD

### CODE REVIEW

**Review by GiCharkviani — APPROVED** [review]

**Review by aweng98 — APPROVED** [review]
LGTM.

File: ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.html [hunk 1]
```diff
@@ -310,7 +310,7 @@ <h4>There are no kit requests</h4>
           </td>
           <td [ngClass]="{'highlight_sample': isResearchSample(kitRequest)}">
             <!--            kit type-->
-            {{kitRequest.kitTypeName}}
+            {{kitRequest.kitTypeDisplayName}}
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.html:313]: The checking in here is not needed

Comment (aweng98) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.html:313]: Will this (not checking) affect legacy participants?

Comment (zyme) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.html:313]: The backend changes are basically doing the "if no display name, then use kit type", so this should work for legacy participants as well.

File: ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.model.ts [hunk 2]
```diff
@@ -30,7 +31,7 @@ export class KitRequest {
       json.receiveDate, json.deactivatedDate, json.deactivationReason, json.participant, json.easypostAddressId, json.nameLabel,
       json.kitLabel, json.express, json.labelDate, json.noReturn, json.externalOrderNumber, json.externalOrderStatus,
       json.preferredLanguage, json.receiveDateString, json.hruid, json.gender, json.collectionDate, json.firstName,
-      json.lastName, json.dateOfBirth, json.shortId, json.receivedBy, json.sequencingRestriction
+      json.lastName, json.dateOfBirth, json.shortId, json.receivedBy, json.sequencingRestriction, json.displayName
```
Comment (zyme) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.model.ts:34]: Should this be `kitTypeDisplayName` or is it okay to keep `displayName` here?

Comment (pegahtah) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.model.ts:34]: It's `displayName` in the json from the backend

**Review by zyme — APPROVED** [review]

**Pull request merged**